__tests__/app.test.js

```diff
@@ -33,3 +34,23 @@ describe("ANY request to invalid path", () => {
    });
  });
 });
+describe("/api", () => {
+ test("should respond with a 200 status code", () => {
+  return request(app).get("/api").expect(200);
+ });
+ test("should respond with an object with all available endpoints on the API", () => {
+  return request(app)
+   .get("/api")
+   .then(({ body }) => {
+    expect(Object.keys(body.endpoints).length).toBe(
```

Comment:
nicely done

Reply:
however this will shout at you if you don't update the the endpoints.json every ticket!